Readability.js

```diff
@@ -615,6 +615,19 @@ Readability.prototype = {
       if (next && next.tagName == "P")
         br.parentNode.removeChild(br);
     });
+
+    // Remove single-cell tables
+    this._forEachNode(this._getAllNodesWithTag(articleContent, ["table"]), function(table) {
+      var tbody = (table.children.length === 1 && table.children[0].tagName === "TBODY") ? table.children[0] : table;
```

Comment:
Nit: `==` rather than `===` please.

Also, I wonder if for 'real' DOM implementations it'd be cheaper to compare `firstElementChild == lastElementChild`. I guess that's longer to write though...

Reply:
Now defers to a generalised version of `_hasSinglePInsideElement` to keep things consistent